### ISSUE TITLE
Parser creates fake tables upon import

### ISSUE THREAD

**Issue by xlearn-pty-ltd** [xlearn-pty-ltd/phpdocx#47]
ref:
https://visaszone.learnifyapp.104.197.211.213.xip.io/lessons/ariva/

doc:
Ariva Sales leaflet March 2017.docx

![image](https://user-images.githubusercontent.com/45555646/51268589-2d430a80-19c9-11e9-88ea-93142b4a6702.png)

![image](https://user-images.githubusercontent.com/45555646/51268625-42b83480-19c9-11e9-92c6-70df4446110e.png)

**Comment by xlearn-pty-ltd** [comment]
I've sent a message to PHPdocx, to see what they say here...

**Comment by xlearn-pty-ltd** [comment]
This issue is random & appears quite isolated, out of the 20 odd test documents, we only found this issue in 2 of them. The PHPdocx team also mention that the parser would only output table elements if a table was found, meaning in the XML markup of the document, there must be some table tags hidden in there - I can't find them in the front end, when using Word, but they exist nonetheless in the XML.

Closing issue for now, until more information is known about this / more examples are found.
